src/transforms/util/pullRequestLinkGenerator.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Generates a create pull request link for GH in the format [URL]/compare/[name]?title=[...keys]-[name]
+ * This format allows the use of title query param to set the PR name
+ */
+export const generateCreatePullRequestUrl = (baseUrl: string, name: string, issueKeys: null | string[] = []) => {
```

Comment:
issueKeys should never be `null` if  you're using a default value for it: `issueKeys: string[] = []`

Reply:
it was to appease typescript for the function issueKeyParser() in src/transforms/pull-request.ts which returns a null | string issuekey set,  but I can handle earlier before this functions called.